prediction_market_agent/agents/replicate_to_omen_agent/omen_resolve_replicated.py

```diff
@@ -79,7 +79,6 @@ def omen_finalize_and_resolve_and_claim_back_all_markets_based_on_others_tx(
         api_keys,
         created_not_claimed_questions,
         auto_withdraw=True,
-        silent_errors=True,
```

Comment:
That's my change, but how come mypy didn't catch it 🤔 Anyway, thank you

Reply:
Oh wait, got it!

PMA is versioned to use ^0.48.6 of PMAT https://github.com/gnosis/prediction-market-agent/blob/main/pyproject.toml#L37

But this change was introduced in 0.48.8 https://github.com/gnosis/prediction-market-agent-tooling/releases

I guess you have now locally installed a new version of PMAT from your other PR?

Reply:
:)))))) no worries!